internal/dataplane/kongstate/types.go

```diff
@@ -43,10 +43,10 @@ func (p *PortDef) CanonicalString() string {
 }
 
 type ServiceBackend struct {
-	Name      string
-	Namespace string
-	PortDef   PortDef
-	Weight    *int32
+	Name      string  `json:"name"`
+	Namespace string  `json:"namespace"`
+	PortDef   PortDef `json:"port_def"`
+	Weight    *int32  `json:"weight"`
```

Comment:
Do we need these tags for something?

Reply:
nope, it's a leftover. I'm gonna remove it